app/views/destinations/edit.html.erb

```diff
@@ -1,9 +1,21 @@
 <%= content_for(:title) { "Editing #{@destination.name_in_database}" } %>
 
-<%= render 'shared/breadcrumb_nav' %>
+<%= content_for :header do %>
+  <%= render 'shared/breadcrumb_nav' %>
 
-<h1><%= @destination.name_in_database %></h1>
+  <h1 class="header__title"><%= @destination.name_in_database %></h1>
```

Comment:
I'm a bit confused, is `header` a block that this is an element of? I don't see the `header` class here

Reply:
Hey @isabel-anastasiadis-boost! Yes the title is an element inside the header. The header is on the application.html.erb where the yield :header is :)